decidim-core/config/locales/en.yml

```diff
@@ -520,6 +526,8 @@ en:
         hero:
           participate: Participate
           welcome: Welcome to %{organization}!
+        metrics:
+          headline: Decidim in numbers
```

Comment:
@decidim/product I guess we need a different text here, not all instances use `Decidim`

Reply:
@oriolgual right, we could use "Participation in figures"